web/translation/translate.en_US.toml

```diff
@@ -77,30 +77,34 @@
 "title" = "Overview"
 "memory" = "RAM"
 "hard" = "Disk"
+"systemInfo" = "System"
+"hostname" = "Hostname"
 "xrayStatus" = "Status"
```

Comment:
It is not clear which status do you mean.
You removed Xray in the main page as well